dietpi/func/dietpi-set_software

```diff
@@ -612,8 +612,8 @@ _EOF_
 		else
 
 			#DietPi-Software PW
-			G_WHIP_YESNO 'Do you want to adjust the default global password for DietPi-Software installations and new unix users? We especially recommend to change the default password "dietpi". This does not effect any existing unix user login.\n
-NB: This password will be saved as plain text within /DietPi/dietpi.txt to be useable by DietPi scripts for e.g. web application and database logins. We highly recommand to adjust all passwords for web services and new unix users independently afterwards.'
+			G_WHIP_YESNO 'Do you want to adjust the default global password for DietPi-Software installations and new unix users? We especially recommend to change the default password "dietpi". This does not affect any existing unix user login.\n
+NB: This password will be saved as plain text within /DietPi/dietpi.txt to be useable by DietPi scripts for e.g. web application and database logins. We highly recommend to adjust all passwords for web services and new unix users independently afterwards.'
```

Comment:
👍, I already fixed an effect != affect somewhere else 🤣, my mistake and bad `e`/`a` mix up, AFAIK. Thanks for fixing.